meinberlin/apps/plans/templates/meinberlin_plans/plan_detail.html

```diff
@@ -76,19 +76,17 @@ <h1 class="item-detail__title u-break-word">
                             <dd>{{ object.reference_number }}, {% trans 'updated on ' %}{% if object.modified %}{{ object.modified|date }}{% else %}{{ object.created|date }}{% endif %}</dd>
                         </dl>
                     </ul>
-                </div>
-
-                <div class="item-detail-2__content">
-                    <div class="item-detail__basic-content">
-                      {% if object.description_image %}
+                    {% if object.description_image %}
                         <div class="item-detail__image--background" style="background-image: url({% thumbnail object.description_image 'item_image' %})">
                         {% if object.description_image_copyright %}
                             <span class="item-detail__copyright copyright">© {{ object.description_image_copyright }}</span>
                         {% endif %}
                         </div>
-                        {% endif %}
-
+                    {% endif %}
+                </div>
```

Comment:
Should this be here? You didn't add an opening div.

Reply:
i doublechecked this and saw that this was already there and i just shifted the `tile image` part  right above it.
Don't know why its marked as new/added in the diff.